packages/devtools_server/lib/src/handlers.dart

```diff
@@ -82,12 +82,12 @@ class ServerApi {
   FutureOr<shelf.Response> logScreenView(shelf.Request request) =>
       notImplemented(request);
 
-  /// Gets a user's LDAP for use in analytics and for distinguishing internal
+  /// Gets a user's id for use in analytics and for distinguishing internal
   /// users from external users.
   ///
   /// This endpoint is not supported externally and will only be implemented for
   /// the version of DevTools used inside Google.
-  FutureOr<shelf.Response> getUserLdap(shelf.Request request) =>
+  FutureOr<shelf.Response> getUserId(shelf.Request request) =>
```

Comment:
this should get getClientId right?  And it's implemented both internally and externally too?

Reply:
We aren't passing the ldap in either internal or external at the moment. The internal api will return a 200 and the external api will return a 204, so that is how we are determining whether a user is internal or not. Are there going to be cases in the future where we want to do something with an internal users LDAP? @jacob314 

If not and if we are only concerned with the isInternal bit, then perhaps we should change this API to be `isInternal`.

Reply:
yes it seems like this isn't the API we want. Adding an api called isInternal suggests that maybe this is the wrong API to be exposing in the first place as each version of devtools should be able to know exactly whether it is internal or not.

Reply:
lets assess if this is the api we want. I'm not sure we actually have a use case for this api as currently written.